docs/parsing_statistics.md

```diff
@@ -12,14 +12,27 @@ TGen is capable of modeling generic behaviors with an action-dependency graph in
 
 ```yaml
 general:
-  # stop after 1 simulated hour
-  stop_time: 3600
+  # stop after 600 simulated seconds
```

Comment:
"stop after 10 minutes" ?

Actually might as well not repeat the number at all, especially now that the unit is in the yaml itself. Maybe just "How long to run the simulation", or drop completely

Reply:
Removed the comment.